_includes/swarm-mode.md

```diff
@@ -0,0 +1,11 @@
+> **Note**
+>
+> Swarm mode is an advanced feature of Docker Engine and Docker Desktop.
+> It's useful for managing a cluster of Docker daemons.
+>
+> Swarm mode is useful if you intend to use Swarm as a production runtime
+> environment. If you're not planning on deploying with Swarm,
+> use [Docker Compose](/compose/) instead.
```

Comment:
This may be a bit confusing because swarm itself is not directly related to compose (or stack deploy), as you may be running `docker service` commands without using `docker stack`.

So perhaps we need to only put this include in pages describing `docker stack`

Reply:
OK, let me make this change

Reply:
I updated now, still including the note on the "index" page for swarm, but removing it from other pages except stack deploy. Also included it under get-started/swarm-deploy.